### PR TITLE
Redesign package version picker

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Summary
Fixes ENG-368.

[Design](https://www.figma.com/file/RGR6BUySI9EfxBXbLGDPMV/Packages?node-id=2%3A2).

Zrzut ekranu 2022-08-10 o 11 53 33 Zrzut ekranu 2022-08-10 o 11 51 39
Zrzut ekranu 2022-08-10 o 11 51 11

## Test Plan
Tested manually.

## Checklist
- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

**Comment by michaeljguarino** [comment]
For some reason I can't click any of the sidebar tabs in the preview link instance, not sure what the cause might be, perhaps some borked click handler

**Review by floreks — APPROVED** [review]

**Comment by floreks** [comment]
I see the same issue when running locally. It seems that the invisible select popover element takes a lot of space and acts as an overlay that also blocks access to other UI elements.

**Comment by maciaszczykm** [comment]
Yep, it looks like `display: none` is missing somewhere and invisible layer blocks access to elements underneath.

File: www/src/components/repos/common/misc.js [hunk 1]
```diff
@@ -93,49 +96,84 @@ export function PackageHeader({ name, icon }) {
...
+                showExtra
+                chips={[
+                  v.id === installed?.version?.id && (
+                    <Chip
+                      severity="success"
+                      size="small"
+                    >
+                      Installed
+                    </Chip>
+                  ),
```
Comment (dogmar) [inline www/src/components/repos/common/misc.js:150]: This inserts `false` as the first element of the array if it's not the installed version. So the wrong number of chips gets displayed.

Comment (dogmar) [reply www/src/components/repos/common/misc.js:150]: This should work: `...(v.id === installed?.version?.id ? [(<Chip />)] : [])`

**Review by dogmar — CHANGES_REQUESTED** [review]

File: www/src/components/repos/common/misc.js (hunk 1 above)
Comment (maciaszczykm) [reply www/src/components/repos/common/misc.js]: Good catch, thanks.

**Review by floreks — APPROVED** [review]

File: www/src/components/repos/common/misc.js [hunk 2]
```diff
@@ -93,49 +96,84 @@ export function PackageHeader({ name, icon }) {
...
+                showExtra
+                chips={[
+                  ...(v.id === installed?.version?.id ? (
+                    <Chip
+                      severity="success"
+                      size="small"
+                    >
+                      Installed
+                    </Chip>
+                  ) : []),
```
Comment (dogmar) [inline www/src/components/repos/common/misc.js:150]: Now this errors if the package is installed. Need to wrap the "Installed" `Chip` with `[]` for the spread operator to work.

**Review by dogmar — CHANGES_REQUESTED** [review]

File: www/src/components/repos/common/misc.js (hunk 2 above)
Comment (maciaszczykm) [reply www/src/components/repos/common/misc.js]: 🤦🏼 Fixed.

**Comment by maciaszczykm** [comment]
@dogmar PTAL.

**Review by michaeljguarino — APPROVED** [review]
Looks great

**Pull request merged**